docs/source/based_features.rst

```diff
@@ -274,3 +274,13 @@ Checked f-strings
     f"{None:0>2}"  # error: The type "None" doesn't support format-specifiers
     f"{date(1,1,1):%}"  # error: Invalid trailing '%', escape with '%%'
     f"{'s':.2f}"  # error: Incompatible types in string interpolation (expression has type "str", placeholder has type "int | float | complex")
+
+Annotations in Functions
+------------------------
+
+Basedmypy handles type annotations in function bodies as unevaluated:
```

Comment:
maybe link to the pep or whatever that explains this, since no one seems to be aware of this functionality